Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/ColorPicker.cpp

```diff
@@ -655,11 +663,21 @@ ColorPicker::ColorPicker(ColorPicker::Configuration configuration, const QString
     // Final color space comment
     m_commentSeparator = makePaddedSeparator(this);
     containerLayout->addWidget(m_commentSeparator);
-    m_commentLabel = new QLabel(QObject::tr("sRGB Float"), this);
+    m_commentLabel = new QLabel(QObject::tr("Color space: sRGB"), this);
     containerLayout->addWidget(m_commentLabel);
 
-    // Place the RGB final input fields below the color space comment
-    containerLayout->addLayout(m_rgbLayout);
+    // Alternate color space info
+
+    m_alternateColorSpaceInfoLayout = new QGridLayout();
+    containerLayout->addLayout(m_alternateColorSpaceInfoLayout);
+
+    // These widgets will be updated and added to m_alternateColorSpaceInfoLayout as needed in setAlternateColorspace* functions
+    m_alternateColorSpaceIntLabel = new QLabel("Alternate Int");
+    m_alternateColorSpaceFloatLabel = new QLabel("Alternate Float");
+    m_alternateColorSpaceIntValue = new QLineEdit("Unspecified");
+    m_alternateColorSpaceFloatValue = new QLineEdit("Unspecified");
```

Comment:
Nit: Wrap these strings in `tr` so that they can be translated since they are user-facing.